ktor-client/ktor-client-core/posix/src/io/ktor/client/HttpClient.kt

```diff
@@ -12,6 +12,8 @@ import io.ktor.utils.io.*
  *
  * The [HttpClientEngine] is selected from the dependencies.
  * https://ktor.io/docs/http-client-engines.html
+ *
+ * [Report a problem](https://ktor.io/feedback/?fqname=io.ktor.client.HttpClient)
```

Comment:
Minor. Should we distinct "fqname" for a class `HttpClient` and a function `HttpClient`?

Reply:
We may include the line number, could you tell me what you think?

Reply:
I think we can solve this problem later, if necessary

Reply:
Sure, I added the line numbers to the lib: https://github.com/e5l/kdoc-annotator